marshmallow/utils.py

```diff
@@ -334,16 +334,31 @@ def callable_or_raise(obj):
     return obj
 
 
-def get_func_args(func):
+def get_func_args_2(func):
     """Given a callable, return a tuple of argument names. Handles
     `functools.partial` objects and class-based callables.
+    This is the orignial Python 2 version
     """
     if isinstance(func, functools.partial):
-        return inspect.getargspec(func.func).args
+        return tuple(inspect.getargspec(func.func).args)
     if inspect.isfunction(func) or inspect.ismethod(func):
-        return inspect.getargspec(func).args
-    # Callable class
-    return inspect.getargspec(func.__call__).args
+        return tuple(inspect.getargspec(func).args)
```

Comment:
No, you are right, it could be a list as well. The previous comment on the function said it returns a tuple, which was somewhat inconsistent, since it returned a list. 
```
"""Given a callable, return a tuple of argument names. Handles
'functools.partial' objects and class-based callables.
"""
So if it returns a list, I think the comment should be made consistent, otherwise this is fine as well.
```